src/parser/whamm.pest

```diff
@@ -57,13 +60,16 @@ TYPE = _{ TY_I32 | TY_BOOL | TY_STRING | TY_TUPLE | TY_MAP }
 // ---- Statements ----
 // ====================
 
-arg = { tuple | expr | val }
+arg = { tuple | expr | val | ternary }
 fn_call = { ID ~ "(" ~ ( arg )? ~ ( "," ~ arg )* ~ ")" }
+block = { "{" ~ (statement)* ~ "}" }
+
 
 // var ops
+return_statement = { "return" ~ expr }
```

Comment:
rename this to `return`?

Reply:
Either way. I like return_statement as it makes it clear its a type of statement